design/schedules.md

```diff
@@ -0,0 +1,76 @@
+# Schedules
+
+We already support `trigger` and (soon) `pr` triggers.
+This spec covers scheduled triggers.
+By way of introduction, here's a rich example of the syntax:
+
+```yaml
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+    - releases/*
+    exclude:
+    - releases/ancient/*
+- cron: "0 12 * * 0"
+  displayName: Weekly Sunday build
+  branches:
+    include:
+    - releases/*
+  batch: true
+  always: true
+```
+
+These definitions say:
+- For `master` and all `releases/*` branches except those under `releases/ancient/*`, run a pipeline at midnight every day.
+- For all `releases/*` branches, run a pipeline at noon on Sundays.
+  - Only run against branches which have changed (the `always` keyword).
```

Comment:
> Only run against branches which have changed

How do you expression that? `always: false` ? What is the default?

Reply:
I messed up the human-readable string here :) It should say "Run even against branches which haven't changed".